doc/whats_new/v1.4.rst

```diff
@@ -926,6 +926,10 @@ Changelog
   `transform` without calling `fit` since `categories` always requires to be checked.
   :pr:`27821` by :user:`Guillaume Lemaitre <glemaitre>`.
 
+- |Enhancement| :class:`preprocessing.LabelEncoder` now supports the
+  `Array API <https://data-apis.org/array-api/latest/>`_. See :ref:`array_api`
+  for more details. :pr:`27381` by :user:`Omar Salman <OmarManzoor>`.
```

Comment:
There is a PR which is already creating the 1.6.rst #28962. So I think let's merge that and then merge main here so that I don't create any unnecessary conflicts. 😄

Reply:
Done.